### ISSUE TITLE
Kite for ipynb support in vs code

### ISSUE THREAD

**Issue by asysuev** [kiteco/issue-tracker#259]
It is really inconvenient, that everything works fine in py files, but in ipynb I lose all possibilities of kite.

**Comment by ryan11lee** [comment]
Would also love to see for Pycharm

**Comment by ldorigo** [comment]
Also hoping to see this soon. A workaround for now is to use a normal python file and surround cells with '#%%' , which allows running them in a python notebook on the side.

**Comment by Jihad** [comment]
ah was hitting my head on why it worked for jupyter in chrome but not vscode.. hope to see it supported soon!

**Comment by witwit** [comment]
This is a must-have feature!!

**Comment by AroMorin** [comment]
I agree, this is a requested feature because I CAN use Kite in JupyterLab and Jupyter Notebooks (not sure about the latter but should work!). So I expected that VS Code would also include Kite support for Jupyter notebook editing. Please implement this :)

**Comment by akashva50** [comment]
I can't integrate kite to jupyter lab|notebook yet. It won't work. I appreciate if someone could help with this.

**Comment by facohen** [comment]
I love Kite and would like to see integration with VS Code running ipynb files.

**Comment by mertgoksel** [comment]
This is a must have for kite. I would love to see it.

**Comment by mh4ey** [comment]
I agree this is a must have feature

**Comment by miwilc** [comment]
still hopefully waiting.

**Comment by MichaelvonSiebenthal** [comment]
Would be great to have kite integration.

**Comment by rockyfjord** [comment]
VS Code suggesting "import pandas as pandas" led me here.

**Comment by chenhajaj** [comment]
+1

**Comment by ParteekSJ** [comment]
Are there any updates?

**Comment by azzzoto** [comment]
Are there any news?

**Comment by hacen-ai** [comment]
Are there any updates?

**Comment by KhalidElsadig** [comment]
any news ?

**Comment by wdenny3885** [comment]
jumping on the bandwagon here. Just finally loaded kite for vscode after seeing it for the 1000,th time on youtube. doesn't work on jupyter in vscode, total let down.

**Comment by arifsudibyo** [comment]
still waiting...

**Comment by Precisionguided** [comment]
same

**Comment by sunilpandey5** [comment]
still waiting

**Comment by kawaranagi** [comment]
also waiting

**Comment by ghost** [comment]
Any updates on this topic? Still waiting.

**Comment by aqlkzf** [comment]
any suggestion on how to use kite for ipynb in vscode?

**Comment by JeroenVanB** [comment]
I would really like this as well

**Comment by chizeni24** [comment]
oh my, nothiong yet?.

**Comment by chizeni24** [comment]
https://www.tabnine.com/welcome

Not a solution but an alternative

**Comment by smdtql** [comment]
Hopefully there will be a solution soon. Seem to me that it is not a diffcult problem.

**Comment by dexin-peng** [comment]
upvote

**Comment by nd-dew** [comment]
i'd highly appreciate this idea

**Comment by nilsschillmann** [comment]
same

**Comment by MateuszSadowski** [comment]
bump

**Comment by Yixi-Rao** [comment]
see you guys next year

**Comment by lasse-meixner** [comment]
Is there a reason for this not existing yet?

**Comment by toihr** [comment]
bump

**Comment by Luizzavala** [comment]
hi, i interest in solution for ipynb, so much thanks

**Comment by ithakis** [comment]
Hi, still waiting on the release. Kite was so good on Pycharm using Jupiter notebooks that I really miss it on VScode.

**Comment by Lunarion1337** [comment]
Bumping this up again
